mcp/src/main/java/io/modelcontextprotocol/server/McpServer.java

```diff
@@ -4,11 +4,7 @@
 
 package io.modelcontextprotocol.server;
 
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```

Comment:
Wildcard imports are discouraged as they can lead to namespace pollution and make it harder to track dependencies.